BonMot.xcodeproj/project.pbxproj

```diff
@@ -948,7 +948,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "if which swiftlint >/dev/null; then\n    swiftlint\nelse\n    echo \"warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint\"\nfi";
+			shellScript = "if which swiftlint >/dev/null; then\n    swiftlint || echo \"warning: SwiftLint has failed…\"\nelse\n    echo \"warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint\"\nfi";
```

Comment:
#nth print the error code, like `SwiftLint failed with exit code ##`
#nth print a recovery suggestion for the most common failure reason: "Is SwiftLint installed and up to date?"
#req remove the ellipsis. There's no reason this can't be a complete sentence. I strongly dislike error messages that just trail off.